lib/dataSchemaHelpers.ts

```diff
@@ -105,8 +105,67 @@ export enum SchemaDataId {
     ProstateCancerTier3 = 'bts:ProstateCancerTier3',
     SarcomaTier3 = 'bts:SarcomaTier3',
+
+    AgeAtDiagnosis = 'bts:AgeatDiagnosis',
+    YearOfDiagnosis = 'bts:YearofDiagnosis',
+    DaysToLastFollowup = 'bts:DaystoLastFollowup',
+    DaysToLastKnownDiseaseStatus = 'bts:DaystoLastKnownDiseaseStatus',
+    DaysToDiagnosis = 'bts:DaystoDiagnosis',
+    PercentTumorInvasion = 'bts:PercentTumorInvasion',
+    GrossTumorWeight = 'bts:GrossTumorWeight',
+    BreslowThickness = 'bts:BreslowThickness',
+    MitoticCount = 'bts:MitoticCount',
+    MarginDistance = 'bts:MarginDistance',
+    TumorDepth = 'bts:TumorDepth',
+    DaysToBirth = 'bts:DaystoBirth',
+
+    CollectionDaysfromIndex = 'bts:CollectionDaysfromIndex',
+    ProcessingDaysfromIndex = 'bts:ProcessingDaysfromIndex',
+    DysplasiaFraction = 'bts:DysplasiaFraction',
+    NumberProliferatingCells = 'bts:NumberProliferatingCells',
+    PercentEosinophilInfiltration = 'bts:PercentEosinophilInfiltration',
+    PercentGranulocyteInfiltration = 'bts:PercentGranulocyteInfiltration',
+    PercentInflamInfiltration = 'bts:PercentInflamInfiltration',
+    PercentLymphocyteInfiltration = 'bts:PercentLymphocyteInfiltration',
+    PercentMonocyteInfiltration = 'bts:PercentMonocyteInfiltration',
+    PercentNecrosis = 'bts:PercentNecrosis',
+    PercentNeutrophilInfiltration = 'bts:PercentNeutrophilInfiltration',
+    PercentNormalCells = 'bts:PercentNormalCells',
+    PercentStromalCells = 'bts:PercentStromalCells',
+    PercentTumorCells = 'bts:PercentTumorCells',
+    PercentTumorNuclei = 'bts:PercentTumorNuclei',
 }
 
+const NUMERICAL_SCHEMA_DATA_LOOKUP: { [schemaDataId: string]: boolean } = {
+    [SchemaDataId.AgeAtDiagnosis]: true,
+    [SchemaDataId.YearOfDiagnosis]: true,
+    [SchemaDataId.DaysToLastFollowup]: true,
+    [SchemaDataId.DaysToLastKnownDiseaseStatus]: true,
+    [SchemaDataId.DaysToDiagnosis]: true,
+    [SchemaDataId.PercentTumorInvasion]: true,
+    [SchemaDataId.GrossTumorWeight]: true,
+    [SchemaDataId.BreslowThickness]: true,
+    [SchemaDataId.MitoticCount]: true,
+    [SchemaDataId.MarginDistance]: true,
+    [SchemaDataId.TumorDepth]: true,
+    [SchemaDataId.DaysToBirth]: true,
+    [SchemaDataId.CollectionDaysfromIndex]: true,
+    [SchemaDataId.ProcessingDaysfromIndex]: true,
+    [SchemaDataId.DysplasiaFraction]: true,
+    [SchemaDataId.NumberProliferatingCells]: true,
+    [SchemaDataId.PercentEosinophilInfiltration]: true,
+    [SchemaDataId.PercentGranulocyteInfiltration]: true,
+    [SchemaDataId.PercentInflamInfiltration]: true,
+    [SchemaDataId.PercentLymphocyteInfiltration]: true,
+    [SchemaDataId.PercentMonocyteInfiltration]: true,
+    [SchemaDataId.PercentNecrosis]: true,
+    [SchemaDataId.PercentNeutrophilInfiltration]: true,
+    [SchemaDataId.PercentNormalCells]: true,
+    [SchemaDataId.PercentStromalCells]: true,
+    [SchemaDataId.PercentTumorCells]: true,
+    [SchemaDataId.PercentTumorNuclei]: true,
+};
```

Comment:
We wouldn't need any of these if `DataSchemaData` has better values for the `type` field. Unfortunately it is almost always `rdfs:Class` which is not very helpful.

Reply:
hmm yeah I think it would be good to fix the schema. @milen-sage is this something that would be easy to add?